spec.html

```diff
@@ -37148,7 +37148,7 @@ <h1>AsyncFromSyncIteratorContinuation ( _result_, _promiseCapability_ )</h1>
           1. Let _valueWrapper_ be PromiseResolve(%Promise%, _value_).
           1. IfAbruptRejectPromise(_valueWrapper_, _promiseCapability_).
           1. Let _steps_ be the algorithm steps defined in <emu-xref href="#sec-async-from-sync-iterator-value-unwrap-functions" title></emu-xref>.
-          1. Let _onFulfilled_ be CreateBuiltinFunction(_steps_, &laquo; [[Done]] &raquo;).
+          1. Let _onFulfilled_ be ! CreateBuiltinFunction(_steps_, &laquo; [[Done]] &raquo;).
```

Comment:
same thing here. `!` is unnecessary and becomes less meaningful to where it actually indicates effect.